src/assets/translations/main.json

```diff
@@ -168,7 +168,9 @@
         "foxyBalance": "FOXy Balance",
         "withdrawals": "Withdrawals",
         "emptyWithdraws": "Pending or active withdrawals will show here.",
-        "availableDate": "available %{date}"
+        "availableDate": "available %{date}",
+        "approveToken": "Approve",
+        "confirm": "Confirm"
```

Comment:
Can you use `common.confirm` for this?

@LPX55 made a [good discussion post](https://github.com/shapeshift/web/discussions/1446) the other day about this - we should reuse common whenever we can, rather that adding new keys.

Reply:
Wasn't sure if `common.approveOnWallet` and `common.confirmOnWallet` was used anywhere else in the platform; will do a quick search and make appropriate changes